lithium/lithium.py

```diff
@@ -141,7 +143,8 @@ def __init__(self):
         self.cutBefore = self.DEFAULT_CUT_BEFORE
 
     def readTestcaseLine(self, line):
-        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + b"]|$|(?=[" + self.cutBefore + b"]))"
+        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + \
+            b"]|$|(?=[" + self.cutBefore + b"]))"
```

Comment:
The trailing ``` +``` seems also to not be required. What do you think, @jschwartzentruber ?

e.g.:
```
print(b"foo"
    b"bar"
    b"2000")
```

still gives `foobar2000`.